proxy/logging/LogHost.h

```diff
@@ -52,13 +52,13 @@ class LogHost
   // preprocess the given buffer data before sent to target host
   // and try to delete it when its reference become zero.
   //
-  bool preproc_and_try_delete(LogBuffer *lb);
+  bool preproc_and_try_delete(LogBuffer *&lb);
```

Comment:
The old signature here used to comply with the LogBufferSink interface. I'm actually slightly surprised to observe that LogHost isn't a LogBufferSink, actually. This "breaks" that compatibility, though it's obviously not a significant issue.